### DISCUSSION TITLE
Camera configuration file

### DISCUSSION THREAD

**Discussion by the0the** [sair-lab/GroundSLAM#8] [Q&A] [answered]
Hello, your work on NI-SLAM is very impressive. May I know where can I get  the camera parameters of your own dataset? Looking forward to your reply, thank you!

**Comment by xukuanHIT** [comment, marked as answer]
@the0the Thanks for your interest. The camera parameters can be found [here](https://entuedu-my.sharepoint.com/:u:/g/personal/kuan_xu_staff_main_ntu_edu_sg/EQqXW2eDcXNAkBWlSGgq6bgBe2yYdjdcuSJ8HEFey3haGg?e=nOg2ym).

**Reply by the0the** [reply]
Thank you very much. I wish you all the best and every success.
